backend/tests/test_config.py

```
@@ -1,17 +1,111 @@
+        "DATABASE_URL", "postgresql+asyncpg://test:test@localhost:5432/test_db"
+    )
+    monkeypatch.setenv("RUNTIME_ENVIRONMENT", "local")
+    monkeypatch.delenv("AUTH_SESSION_HMAC_SECRET", raising=False)
+
+    loaded_settings = _settings_without_env_file()
+
+    assert loaded_settings.AUTH_SESSION_HMAC_SECRET is None
+
+
+def test_settings_repr_redacts_auth_session_hmac_secret(monkeypatch):
+    secret = "naruon-session-hmac-token-32-byte-minimum"
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "secret"

(S105)